composer/callbacks/memory_monitor.py

```diff
@@ -108,15 +109,8 @@ def after_train_batch(self, state: State, logger: Logger):
 
 
 def _get_memory_report() -> Dict[str, Union[int, float]]:
-    if not torch.cuda.is_available():
-        log.debug("Cuda is not available. The memory report will be empty.")
```

Comment:
from @A-Jacobson: 

> same as my previous comment, I consider this a breaking change.

Reply:
I updated the caller function emit a warning, so I don't think another statement here is necessary.